assets/src/scss/new-identity/style.scss

```diff
@@ -33,4 +33,19 @@
   --author-block--background: var(--beige-100);
   --author-block-description-button--color: var(--gp-green-800);
   --author-block-description-button--hover--color: var(--gp-green-800);
+  --link--color: var(--grey-900);
+  --link--visited--color: var(--grey-900);
+  --link--hover--color: var(--grey-900);
+}
+
+a.external-link,
+a.pdf-link {
+  color: var(--grey-900) !important;
+  text-decoration: none !important;
+  transition: 0.2s cubic-bezier(0.5, 0, 0, 1);
+  border-bottom: 2px solid var(--gp-green-500);
+
+  &:hover {
+    box-shadow: inset 0 -22px var(--gp-green-100);
```

Comment:
Checking the value of `offset-y`, currently set to `-22px`, I see that a bit inconsistent if you check that on screens since the different font sizes we have (see [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_typography.scss#L11-L19)). You can also make that test and see that on small screens there is more gap over the text, but on large screens it's almost the same height.

Can we change the `offset-y` more dynamically? Also if the user override the font-size because of `font-size: var(--text--x-large-and-up--font-size, 1.25rem);`

If you think it's a bit complicated change we can follow up it afterwards and don't block the merge! 🙏 

What do you think?

Reply:
Great idea, we can look into doing that dynamically. If you have any suggestions I will love to hear them.